src/utils.c

```diff
@@ -0,0 +1,26 @@
+#include <string.h>
+#include "../include/utils.h"
+
+Command parseStringToCommand(char commandAsString[]) {
+    char str[strlen(commandAsString)];
+    strcpy(str, commandAsString);
+    char *delimiter = " ";
+    char *token;
+    Command command = {"", "", ""};
+    token = strtok(str, delimiter);
```

Comment:
The [documentation](https://linux.die.net/man/3/strtok) says, that `strtok` is not thread-safe(under Bugs). `strtok_r()` should be used instead.